File: Scripts/utils/entity_utils.lua
local entityUtils = { }

function entityUtils.IsEntityAtCoords(entity, targetX, targetY, targetZ, radius)
    local coords = ENTITY.GET_ENTITY_COORDS(entity, true, true)
        
    -- Check if the entity is within the specified area
    if math.abs(coords.x - targetX) <= radius and math.abs(coords.y - targetY) <= radius and math.abs(coords.z - targetZ) <= radius then
        return true
    end
    
    return false
end

return entityUtils

File: Scripts/utils/entity_utils.py
def is_entity_at_coords(entity, target_x, target_y, target_z, radius):
    coords = ENTITY.GET_ENTITY_COORDS(entity, True, True)
        
    # Check if the entity is within the specified area
    if (abs(coords[0] - target_x) <= radius and abs(coords[1] - target_y) <= radius and abs(coords[2] - target_z) <= radius):
        return True
        
    return False